client/client.go

```diff
@@ -30,24 +32,21 @@ func (e StatusError) Error() string {
 // Client represents a HTTP client for making requests to the FullStory API.
 type Client struct {
 	HTTPClient *http.Client
-	Config
-}
-
-// Config is configuration for Client.
-type Config struct {
-	APIToken string
-	BaseURL  string
+	Config     *config.Config
+	BaseURL    string
 }
 
 // NewClient returns a Client initialized with http.DefaultClient and the
 // supplied apiToken.
-func NewClient(apiToken string) *Client {
+func NewClient(config *config.Config) *Client {
+	baseURL := BaseURL
+	if config.ExportURL != "" {
+		baseURL = config.ExportURL
+	}
 	return &Client{
 		HTTPClient: http.DefaultClient,
-		Config: Config{
-			APIToken: apiToken,
-			BaseURL:  BaseURL,
-		},
+		Config:     config,
+		BaseURL:    baseURL,
```

Comment:
I feel like we shouldn't have both here. If the config already contains a URL, we should use that, and have the default populate when loading the config.

Reply:
Yep, that's way better now that the client code has been moved into the rep. Will update to that.